README.md

```diff
@@ -22,11 +22,19 @@ If you maintain a JavaScript library that has UI components, we'd love for you t
 ### Building
 
 Once you've cloned Chassis to your machine, run these commands:
-```
+```bash
 $ npm install
 $ grunt build
 ```
 
+### Develop
+
+Now that you have the environment ready, you may want to 'watch' your build task, so you can run:
```

Comment:
Reword along the lines of "To automate the build process during development, run the 'watch' task to compile SCSS upon file change."